source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md

```diff
@@ -1,49 +1,45 @@
 
 It only takes a couple minutes to start streaming email event data from SendGrid to Keen IO and once you do you can start analyzing and visualizing your data in a whole bunch of different ways.
 
-## Send your SendGrid Data to Keen IO
+{% anchor h3 %}
+Sending your SendGrid Data to Keen IO
+{% endanchor h3 %}
 
-**Step 1: Create a Keen IO account**
+{% anchor h4 %}
+Step 1: Sign up for Keen IO
+{% endanchor h4 %}
 
-* Create a <a href="https://keen.io/signup">Keen IO account</a> (if you don't already have one)
+Create a <a href="https://keen.io/signup">Keen IO account</a> (if you don't already have one)
 
+{% anchor h4 %}
+Step 2: Activate SendGrid's Event Notification App
+{% endanchor h4 %}
 
-**Step 2: Activate SendGrid's Event Notification App**
-
-First click on <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a> within Settings in the left column. Next, locate and activate the Event Notification Setting.
-
-![SendGrid- Mail Settings]({{root_url}}/images/keen/sendgrid_mailsettings_eventnotification.png)
-
-Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY
-```
-
-If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY
-
- ```
-
-In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
+1. First click on <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a> within Settings in the left column. 
+2. Next, locate and activate the Event Notification Setting.
+   ![SendGrid- Mail Settings]({{root_url}}/images/keen/sendgrid_mailsettings_eventnotification.png)
+3. Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
+    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
+4. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
```

Comment:
The quotes don't really match our style guide. We should decide how to handle Field names/ UI text like this. My suggestion would be:

To do this, check **Select All** under *Select Actions*,